### PR TITLE
fix(web): General petitions web view update

### PR DESCRIPTION
**Pull request by albinagu**
Screenshot 2023-05-22 at 11 37 14

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by kksteini — APPROVED** [review]
LGTM

**Review by RunarVestmann — COMMENTED** [review]
I'm wondering whether we'd like to do something specific if for example a user types in a random id in the url
image

Also if I click on the "Setja nafn mitt á þennan lista" I get an error if the list id isn't valid, could we do something about that :/ 
image

File: apps/web/screens/PetitionView/PetitionView.tsx [hunk 1]
```diff
@@ -253,4 +258,31 @@ const PetitionView = ({ namespace }: PetitionViewProps) => {
   )
 }
 
+interface InitialProps {
```
Comment (RunarVestmann) [inline apps/web/screens/PetitionView/PetitionView.tsx:281]: You could type the component as a Screen instead of creating this InitialProps interface

Comment (RunarVestmann) [reply apps/web/screens/PetitionView/PetitionView.tsx]: I commited that change with your permission :)

**Review by RunarVestmann — APPROVED** [review]

**Pull request merged**